src/content/devices/multi/index.html

```diff
@@ -0,0 +1,82 @@
+ *  tree.
+-->
+<html>
+<head>
+
+  <meta charset="utf-8">
+  <meta name="description" content="WebRTC code samples">
+  <meta name="viewport" content="width=device-width, user-scalable=yes, initial-scale=1, maximum-scale=1">
+  <meta itemprop="description" content="Client-side WebRTC code samples">
+  <meta itemprop="image" content="../../../images/webrtc-icon-192x192.png">
+  <meta itemprop="name" content="WebRTC code samples">
+  <meta name="mobile-web-app-capable" content="yes">
+  <meta id="theme-color" name="theme-color" content="#ffffff">
+
+  <base target="_blank">
+
+  <title>getUserMedia: output device selection</title>
+
+  <link rel="icon" sizes="192x192" href="../../../images/webrtc-icon-192x192.png">
+  <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="../../../css/main.css">
+  <link rel="stylesheet" href="css/main.css">
+
+</head>
+
+<body>
+
+  <div id="container">
+
+    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>getUserMedia: output device selection</span></h1>
+
+    <div id="sources">
+
+     <div>
+       <h2>getUserMedia():</h2>
+       <video class="gum" title="From getUserMedia()" autoplay controls></video>
+       <audio class="gum" title="From getUserMedia()" controls></audio>
+     </div>
+
+     <div>
+       <h2>Local media files:</h2>
+       <video id="localVideo" title="From local video file" controls loop poster="images/poster.jpg" preload="metadata">
+         <source src="video/chrome.webm" type="video/webm" />
+         <source src="video/chrome.mp4" type="video/mp4" />
+         <p>This browser does not support the video element.</p>
+       </video>
+       <audio title="From local audio file" id="localAudio" controls loop>
+         <source src="audio/audio.mp3" type="audio/mp3" />
+         This browser does not support the audio element.
+       </audio>
+     </div>
+
+   </div>
+
+   <p>Press play to select a source. Only one output can be selected for all sources.</p>
+
+   <div>
+    <label>Select audio output: </label>
+    <select id="audioOutput"></select>
+  </div>
+
+  <p>This demo must be run over HTTPS on Chrome (or Chrome Canary) 45.0.2441.x or later.<p>
+  <p><strong>Enable experimental Web Platform features</strong> must be enabled from the chrome://flags page or run Chrome with the following flag:</p>
+  <p>&nbsp;&nbsp;&nbsp;&nbsp;<code>--enable-blink-features=EnumerateDevices,AudioOutputDevices</code></p>
+  <p>More information about running Chrome with flags is available from <a href="https://www.chromium.org/developers/how-tos/run-chromium-with-flags" title="The Chromium Projects: Run Chromium with flags">The Chromium Projects</a>.</p>
+
+  <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/devices/multi" title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
```

Comment:
@KaptenJansson see #546. (Also includes a couple of commits that should have been pushed before I merged this PR :^\...)